Lombiq.UIKit.Widgets/Manifest.cs

```diff
@@ -0,0 +1,9 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Lombiq UI Kit Widgets",
+    Author = "Lombiq Technologies",
+    Version = "0.0.1",
+    Description = "Module for reusable widgets based on Lombiq UI Kit.",
+    Website = "https://github.com/Lombiq/Orchard-UIKit"
+)]
```

Comment:
Add the following dependencies: Lombiq.UIKit (use an appropriate const from the module), OrchardCore.ContentFields, OrchardCore.Media.